.github/workflows/tests.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
 
       - name: Run PyTest
         run: |
-          pytest deepcell_label --cov deepcell_label --flake8
+          pytest deepcell_label --cov deepcell_label
```

Comment:
Clarification: No need to use --flake8 because it is linted and commited before this runs?

Reply:
Flake8 now runs in parallel in the pre-commit hook action.

If the author happens to have used pre-commit to set up the pre-commit hooks with `precommit install` then the pre-commit hook does nothing, but if they have not installed the hooks, then the action runs and commits the hooks for them.